statsmodels/tools/pca.py

```diff
@@ -738,25 +740,31 @@ def plot_rsquare(self, ncomp=None):
         ncomp : int, optional
             Number of components ot include in the plot.  It None, will
             plot the minimum of 10 or the number of computed components
+        ax : Matplotlib axes instance, optional
+            An axes on which to draw the graph.  If omitted, new a figure
+            is created
 
         Returns
         -------
         fig : figure
             Handle to the figure
         """
-        plt.figure()
+        import statsmodels.graphics.utils as gutils
+        import matplotlib.pyplot as plt
+
+        fig, ax = gutils.create_mpl_ax(ax)
+
         ncomp = 10 if ncomp is None else ncomp
         ncomp = min(ncomp, self._ncomp)
         # R2s in rows, series in columns
         r2s = 1.0 - self._ess_indiv / self._tss_indiv
         r2s = r2s[1:]
         r2s = r2s[:ncomp]
-        bp = plt.boxplot(r2s.T)
-        ax = bp['boxes'][0].get_axes()
+        plt.boxplot(r2s.T)
```

Comment:
my mistake for not seeing the plt import
however, the boxplot needs to use the ax instance  (use the object oriented interface so we don't need pyplot)